requirements.txt

```diff
@@ -0,0 +1,4 @@
+mkdocs
```

Comment:
Can we provide specific version like here: https://github.com/Dasharo/docs/blob/master/requirements.txt ?
Can be newer ones, if exist

Reply:
This helps to avoid cases when we render locally with one set of versions, and build in the CI with another set of versions. We've had such problems in the past.

Reply:
Added newest versions.